docs/Database_Schema.md

```diff
@@ -30,6 +30,7 @@ CREATE TABLE IF NOT EXISTS "blocks" (
     "signature"             BLOB    NOT NULL,
     "tx_count"              INTEGER NOT NULL,
     "enrollment_count"      INTEGER NOT NULL,
+    "time_offset"           INTEGER NOT NULL,
     "time_stamp"            INTEGER NOT NULL,
```

Comment:
is this the same as the new `genesis_timestamp` ?

Reply:
equals the time_offset of the blockheader

Reply:
oh I meant the `time_stamp` which is below the `time_offset`

Reply:
The time_stamp is the block's timestamp. it's not genesis_timestamp.

Reply:
Oh got it. Thank you.